src/test/java/uk/gov/justice/digital/delius/controller/secure/CustodyController_updateNomsNumberTest.java

```diff
@@ -0,0 +1,109 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.module.mockmvc.RestAssuredMockMvc;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import uk.gov.justice.digital.delius.controller.advice.SecureControllerAdvice;
+import uk.gov.justice.digital.delius.data.api.*;
+import uk.gov.justice.digital.delius.service.CustodyService;
+import uk.gov.justice.digital.delius.service.OffenderIdentifierService;
+
+import static io.restassured.module.mockmvc.RestAssuredMockMvc.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+public class CustodyController_updateNomsNumberTest {
+
+    private CustodyService custodyService = mock(CustodyService.class);
```

Comment:
I thought in java land using the `mock` like this without the `RunWith` meant that the unnecessary stubbings etc. didn't run

Reply:
although you're using rest assured so :man_shrugging: